pepper-apis/src/test/java/org/broadinstitute/ddp/route/VerifyInvitationRouteTest.java

```diff
@@ -71,7 +71,8 @@ public void testVerificationOfGoodInvitation() throws Exception {
                 HttpStatus.SC_OK, response.returnResponse().getStatusLine().getStatusCode());
 
         TransactionWrapper.useTxn(handle -> {
-            InvitationDto requeriedInvitation = handle.attach(InvitationDao.class).findByInvitationGuid(invitation.getGuid()).get();
+            InvitationDto requeriedInvitation = handle.attach(InvitationDao.class)
```

Comment:
It's probably a lot of work to setup test infrastructure to also boot housekeeping, block for a bit, and then look for evidence that the event has been processed, but I wonder if there's something that we can do with a mock/spy to verify that `EventService.processAllActionsForEventSignal()` has been called for the given signal?

Reply:
I guess one way might be to indirectly test it by explicitly adding event configuration that hooks into the signal trigger type, something simply and verifiable. So in a way we check that it indeed triggers "downstream" events.

Reply:
Updated the tests to do this.